mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.js

```diff
@@ -22,10 +22,16 @@ import { getUUID } from '../../common/utils/ActionUtils';
 export const CHART_TYPE_LINE = 'line';
 export const CHART_TYPE_BAR = 'bar';
 
+// Polling interval
+export const METRICS_PLOT_POLLING_INTERVAL_MS = 5000;
+// Stop polling when the polling duration exceeds this value
+export const METRICS_PLOT_POLLING_DURATION_MS = 3600 * 1000; // 1 hour
```

Comment:
@dbczumar I remember we discussed that we should set an appropriate polling threshold when a run never ends, but I don't think we need to because runs usually end.

Reply:
Offline discussion: check the timestamp of the last metric, and if it's more than 1 week, then we won't refresh.